uaclient/api/u/pro/security/fix/_common/__init__.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, value: int, msg: str):
     SYSTEM_NOT_AFFECTED = _Value(0, "not-affected")
     SYSTEM_STILL_VULNERABLE = _Value(1, "still-affected")
     SYSTEM_VULNERABLE_UNTIL_REBOOT = _Value(2, "affected-until-reboot")
+    SYSTEM_VULNERABLE = _Value(3, "affected")
```

Comment:
The rationale for having this as a different message was that if the user would like to see if their system is affected, a `still-affected` status could be confusing compared to a simple binary - `affected` / `not-affected`. Also, the `still-affected` message is used to convey the expected_status where it makes sense since we provide with a reason as to why the fix/plan did not resolve the cve.

But I understand if having this only adds to complexity and can be substituted by the already existing still-affected message. What do you suggest @orndorffgrant ?